proposals/lip-0060.md

```diff
@@ -100,7 +100,11 @@ The asset schema is the same as defined in [LIP 0055][lip-0055#asset-schema].
 The block assets property is validated as follows:
 
 * _Static validation of the genesis block_:
```

Comment:
I think it would be nice to still mention:

> In contrast to the block processing in [LIP 0055][lip-0055#asset-validation], for the genesis block there is no limitation on the size of the `data` property of every entry.

Reply:
done